### PR TITLE
Conformance tests for invalid hashes

### PR DESCRIPTION
**Pull request by dnadales**
Part of https://github.com/input-output-hk/cardano-ledger-specs/issues/787

### CODE REVIEW

File: cardano-ledger/test/Test/Cardano/Chain/Block/Model.hs [hunk 1]
```diff
@@ -260,11 +260,7 @@ ts_prop_invalidDelegationCertificatesAreRejected =
 
             -- This chooses with even probability between manually tweaked
             -- DCerts and goblin-tweaked ones.
-            invalidDelegationCerts = Gen.choice
-              [ Gen.list (Range.constant 0 10)
-                         (randomDCertGen delegationEnv)
-              , tamperedDcerts delegationEnv delegationSt
-              ]
+            invalidDelegationCerts = tamperedDcerts delegationEnv delegationSt
```
Comment (mhuesch) [inline cardano-ledger/test/Test/Cardano/Chain/Block/Model.hs:263]: 👍

**Review by mhuesch — APPROVED** [review]
LGTM 👍

**Comment by dnadales** [comment]
bors retry

**Pull request merged**